Update bulk-add-custodians.md (#8572)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/bulk-add-custodians.md

## Before you import custodians

> - You can import a maximum of 1,000 custodians (rows) per CSV file. [Note: Importing 1000 custodians at once may result in some timeout errors and some custodians may fail the import. To remediate this please repeat the import and the failed custodians should be imported. To avoid timeouts it's recommended to import 200 custodians at a tiem.]

Reviewer: ```suggestion
- You can import a maximum of 1,000 custodians (rows) per CSV file. Note that importing 1000 custodians at the same time might result in timeout errors and some custodians might fail the import. To remediate this, repeat the import and the failed custodians should be imported. To avoid timeouts we recommend importing 200 custodians at a time.]
```

> - You can import a maximum of 1,000 custodians (rows) per CSV file. [Note: Importing 1000 custodians at once may result in some timeout errors and some custodians may fail the import. To remediate this please repeat the import and the failed custodians should be imported. To avoid timeouts it's recommended to import 200 custodians at a tiem.]

Reviewer: ```suggestion
- You can import a maximum of 1,000 custodians (rows) per CSV file. Note that importing 1,000 custodians at the same time might result in timeout errors and some custodians might fail the import. To remediate this, repeat the import and the failed custodians should be imported. To avoid timeouts we recommend importing 200 custodians at a time.
```
